pkg/antctl/transform/common/transform.go

```diff
@@ -26,12 +32,69 @@ type GroupMemberPod struct {
 	IP string `json:"ip,omitempty" yaml:"ip,omitempty"`
 	// Ports maintain the named port mapping of this Pod.
 	Ports []networkingv1beta1.NamedPort `json:"ports,omitempty" yaml:"ports,omitempty"`
+	// PrintName is the name of pods shown in table outputs of commands
+	PrintName string `json:"printname,omitempty" yaml:"printname,omitempty"`
 }
 
 func GroupMemberPodTransform(pod networkingv1beta1.GroupMemberPod) GroupMemberPod {
 	var ipStr string
 	if len(pod.IP) != 0 {
 		ipStr = net.IP(pod.IP).String()
 	}
-	return GroupMemberPod{Pod: pod.Pod, IP: ipStr, Ports: pod.Ports}
+	podName := ""
+	if len(ipStr) > 0 {
+		podName = ipStr
+	} else {
+		podName = pod.Pod.Namespace + "/" + pod.Pod.Name
```

Comment:
We should let AddressGroup and AppliedToGroup to build the content of its members themselves, not putting code in common path based on assumption.
I think PrintName is no longer needed in that way.